README.md
> **使用微光 Lowlight 的服务，即代表您已知悉并同意[使用条款](tos)。**
>
> 本站目前处于公开测试运营阶段，欢迎填写[问卷调查](https://wj.qq.com/s2/1091099/5c62)来支持我们的工作。

# 微光 Lowlight  - 提供一些音游下载的小站

「Lowlight - A Faint Light」

移动端用户点开左上角的菜单按钮就可以看到其他东西了。

本页默认只提供 Android （安卓）安装包，您可前往[桌面端专页](desktop)查看本站收录的部分桌面端游戏。

催更、请求音游收录、提建议、~~骂我~~：[bilibili:微光Lowlight](https://space.bilibili.com/319171871) 

站娘由 [Kuriko](https://space.bilibili.com/7847180) 绘制（

## Arcaea

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **Arcaea 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 7+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 7
>
> App Store 年龄分级 : 4+

</details>

<!-- tabs:start -->

#### **中国大陆版本**

<details>
<summary><b><font color="#8e4bbc">详细信息</font></b></summary>

?> 版本号后含有 c 的是 lowiro 提供的中国大陆版本。该版本没有接入 Google Play 的有关服务，对中国大陆玩家更加友好。同时，该版本[可以直接通过支付宝充值记忆源点](purchase)。您可以直接覆盖 Play 版本来安装本版本，而无需事先卸载。

</details>

### 3.8.8c (2021/10/21)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/arcaea/arcaea_3.8.8c.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/arcaea/arcaea_3.8.8c.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/arcaea/arcaea_3.8.8c.apk)

#### **Play 版本**

<details>
<summary><b><font color="#8e4bbc">需要 SAI 安装器安装，点击查看</font></b></summary>

?>  Play 版本需要通过 Play 商店充值记忆源点，同时部分地区可能无法使用 Play Games 服务。如果您下载的文件为 .xapk/.apks 格式，请参照[这里](sai)进行安装。

</details>

### 3.8.8 (2021/10/21)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/arcaea/arcaea_3.8.8.apks)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/arcaea/arcaea_3.8.8.apks)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/arcaea/arcaea_3.8.8.apks)

#### **Arcaea 官网**

> 如果你需要从官网进行下载，请注意使用中国大陆 IP，也就是说，不要挂代理。

### lowiro limited

[Arcaea 官网上的 Arcaea](https://arcaea.lowiro.com)

#### **愚人节**

### 3.5.3c (2021/04/01|愚人节)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/arcaea/arcaea_3.5.3c.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/arcaea/arcaea_3.5.3c.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/arcaea/arcaea_3.5.3c.apk)

<!-- tabs:end -->

## Lanota

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **Lanota 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> App Store 年龄分级 : 4+

</details>

### 2.6.4(2021/10/20)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/lanota/lanota_2.6.4.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/lanota/lanota_2.6.4.apk)

## Orzmic

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **Orzmic 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3

</details>

<!-- tabs:start -->

#### **TapTap**

### TapTap Games

[TapTap 上的 Orzmic](https://www.taptap.com/app/194778)

#### **其他版本**

### 1.8.22 (2021/08/21) - 最后一个未添加年龄鉴别的版本

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/orzmic/orzmic_1.8.22.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/orzmic/orzmic_1.8.22.apk)

<!-- tabs:end -->

## Phigros

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **Phigros 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 12+
>
> ESRB（美洲|娱乐软件分级委员会）: Teen（青少年）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 12
>
> App Store 年龄分级 : 12+

</details>

<!-- tabs:start -->

#### **TapTap**

### TapTap Games

[TapTap 上的 Phigros](https://www.taptap.com/app/165287)

#### **应用宝**

### Tencent MyApp

[腾讯应用宝上的 Phigros](https://sj.qq.com/myapp/detail.htm?apkName=com.PigeonGames.Phigros)

#### **愚人节**

### 1.6.5 (2021 愚人节)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/phigros/phigros_1.6.5.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/phigros/phigros_1.6.5.apk)

### 1.4.1 (2020 愚人节)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/phigros/phigros_1.4.1.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/phigros/phigros_1.4.1.apk)

#### **其他版本**

### 1.6.11 (2021/07/25) - 最后一个未添加年龄鉴别的版本

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/phigros/phigros_1.6.11.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/phigros/phigros_1.6.11.apk)

### 1.4.3 (2020/05/18) - 可以无限刷 MB 的版本

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/phigros/phigros_1.4.3.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/phigros/phigros_1.4.3.apk)

### 1.0.0 (2019/08/28) - 初版

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/phigros/phigros_1.0.0.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/phigros/phigros_1.0.0.apk)

<!-- tabs:end -->

## TAKUMI³

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **TAKUMI³ 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> App Store 年龄分级 : 4+

</details>

### 3.57 (2021/10/08)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/takumi/TAKUMI³_3.57.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/takumi/TAKUMI³_3.57.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/takumi/TAKUMI³_3.57.apk)

## Cytoid

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> 由于 Cytoid 没有正式上架，故没有可用的分级资讯。
>
> 本站建议您或您的监护人自行确认该游戏的适用年龄。

</details>

### 2.0.2(2020/10/28)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/cytoid/cytoid_2.0.2.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/cytoid/cytoid_2.0.2.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/cytoid/cytoid_2.0.2.apk)

## Dynamite

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> 由于 Dynamite 没有正式上架，故没有可用的分级资讯。
>
> 本站建议您或您的监护人自行确认该游戏的适用年龄。

</details>

### 1.0.1 Rev10

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/dynamite/Dynamite.1.0.1.rev10.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/dynamite/Dynamite.1.0.1.rev10.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/dynamite/Dynamite.1.0.1.rev10.apk)

## Malody

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> 由于 Malody 是由玩家社区驱动的游戏，故无法单纯从其提交的分级信息中判断真实的适用年龄。
>
> 本站建议您或您的监护人自行确认该游戏的适用年龄。

</details>

<!-- tabs:start -->

#### **Android**

### 4.3.7

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/malody/Malody_Andoid_4.3.7.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/malody/Malody_Andoid_4.3.7.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/malody/Malody_Andoid_4.3.7.apk)

#### **Windows**

[前往桌面端专页查看](/desktop?id=malody)

<!-- tabs:end -->

## osu! Lazer

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> 由于 osu! Lazer 没有正式上架，故没有可用的分级资讯。
>
> 本站建议您或您的监护人自行确认该游戏的适用年龄。

</details>

<!-- tabs:start -->

#### **Android**

### 2021.1016.0

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/osulazer/sh.ppy.osulazer_2020.10.16.0.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/osulazer/sh.ppy.osulazer_2020.10.16.0.apk)


#### **桌面端**

[前往桌面端专页查看](/desktop?id=osu)

<!-- tabs:end -->

## Sonolus

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> 由于 Sonolus 没有正式上架，故没有可用的分级资讯。
>
> 本站建议您或您的监护人自行确认该游戏的适用年龄。

</details>

### 0.5.8 Alpha

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/sonolus/Sonolus_0.5.8.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/sonolus/Sonolus_0.5.8.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/sonolus/Sonolus_0.5.8.apk)

## BanG Dream! 少女乐团派对！(ガルパ/GBP/~~邦邦~~)

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **BanG Dream! GBP 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> CADPA（中国大陆|中国数字音像协会）：8+
>
> GSRR（中国台湾|游戏软体分级管理办法）： 普遍级
>
> CERO（日本|计算机娱乐分级机构）： B（12 岁以上）
>
> App Store 年龄分级 : 4+

</details>

<details>
<summary><font color="#8e4bbc"><b>资源引用说明</b></font></summary>

?> 日服 GBP 游戏资源转载自 https://bangdream.fun ，并将资源存放在本站服务器。你也可以前往该站，下载其他区服版本的 GBP。

</details>

<!-- tabs:start -->

#### **日服**

### 5.6.4 (2021/10/22)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/bandori/jp.co.craftegg.band_5.6.4.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/bandori/jp.co.craftegg.band_5.6.4.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/bandori/jp.co.craftegg.band_5.6.4.apk)

#### **简中服**

### bilibili Games

[哔哩哔哩游戏上的梦想协奏曲！少女乐团派对！](https://game.bilibili.com/bangdream/)

<!-- tabs:end -->

## D4DJ Groovy Mix

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **D4DJ Groovy Mix 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> App Store 年龄分级 : 4+

</details>
<details>
<summary><b><font color="#8e4bbc">D4DJ 配置要求说明</font></b></summary>

> 根据 Play Store 页面的要求，您的设备需要满足以下条件：
>
> - 最低兼容：Android 5.0 以上；
> - 推荐设备：处理器性能相当于 Snapdragon 835 或更强的设备（注：2018 年以后的机型大多满足该条件）。

</details>

<!-- tabs:start -->

#### **日服**

<details>
<summary><b><font color="#8e4bbc">需要 SAI 安装器安装，点击查看</font></b></summary>

> 如果您下载的文件为 .xapk/.apks 格式，请参照[这里](sai)进行安装。

</details>

### 3.0.0 (2021/10/24) 

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/d4dj/com.bushiroad.d4dj_2.5.6.xapk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/d4dj/com.bushiroad.d4dj_2.5.6.xapk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/d4dj/com.bushiroad.d4dj_2.5.6.xapk)

#### **国际服 | 英语**

### 2.5.3 (2021/10/21)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/d4dj/com.bushiroad.en.d4dj_2.5.3.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/d4dj/com.bushiroad.en.d4dj_2.5.3.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/d4dj/com.bushiroad.en.d4dj_2.5.3.apk)

<!-- tabs:end -->

## 偶像梦幻祭 Music！(偶像梦幻祭2/あんスタ/ES2)

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **Ensemble Stars!! 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> CAPDA（中国大陆|中国数字音像协会）：12+
>
> App Store 年龄分级 : 4+

</details>

<!-- tabs:start -->

#### **日服**

<details>
<summary><b><font color="#8e4bbc">需要 SAI 安装器安装，点击查看</font></b></summary>

> 如果您下载的文件为 .xapk/.apks 格式，请参照[这里](sai)进行安装。

</details>
<details>
<summary><b><font color="#8e4bbc">ES2 配置要求说明</font></b></summary>

> 根据 Play Store 页面的要求，您的设备需要满足以下条件：
>
> - 最低兼容：Android 5.0 以上（推荐 Android 6.0 以上）；
> - 推荐设备：处理器性能相当于 Snapdragon 835 或更强的设备（注：2018 年以后的机型大多满足该条件）；
> - **如果设备的 USB 调试设置为开启，游戏将无法启动，需要关闭 USB 调试**；
> - **被监测到 root 或非法修改的设备无法启动游戏**。

</details>

### 1.0.56(2021/10/04)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/es2/es2_1.0.56.xapk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/es2/es2_1.0.56.xapk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/es2/es2_1.0.56.xapk)

#### **简中服**

### Happy Elements

[乐元素上的偶像梦幻祭2](https://es2.happyelements.cn/)

<!-- tabs:end -->

## LoveLive! School idol festival (LLSIF/SIF)

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **LoveLive! SIF 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> CADPA（中国大陆|中国数字音像协会）：12+
>
> App Store 年龄分级 : 4+

</details>

<details>
<summary><font color="#8e4bbc"><b>资源引用说明</b></font></summary>

?> 日服 SIF 游戏资源转载自 https://jp.llsif.cn/ ，并将资源存放在本站服务器。你也可以前往该站，下载其他修改版本的 SIF，包括共存修改版及 Root 版等。

</details>

<!-- tabs:start -->

#### **日服**

<details>
<summary><b><font color="#8e4bbc">需要 SAI 安装器安装，点击查看</font></b></summary>

> 如果您下载的文件为 .xapk/.apks 格式，请参照[这里](sai)进行安装。

</details>

### 9.3 (2021/10/11)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/llsif/klb.android.lovelive_9.3.xapk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/llsif/klb.android.lovelive_9.3.xapk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/llsif/klb.android.lovelive_9.3.xapk)

####  **简中服**

### Shengqu Games

[盛趣游戏上的 LoveLive！学园偶像祭](https://ll.sdo.com)

#### **国际服 | 英语**

### 9.3 (2021/10/11)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/llsif/klb.android.lovelive_en_9.3.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/llsif/klb.android.lovelive_en_9.3.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/llsif/klb.android.lovelive_en_9.3.apk)

<!-- tabs:end -->

## 世界计划 多彩舞台(プロセカ/PRSK/PJSK/~~啤酒烧烤~~)

<details>
<summary><font color="#8e4bbc"><b>未成年人保护资讯</b></font></summary>

> **Project Sekai 可供参考的分级认证信息如下：**
>
> IARC（国际年龄评级联盟系统）: 3+
>
> ESRB（美洲|娱乐软件分级委员会）: Everyone（适合所有人）
>
> PEGI（欧洲|泛欧洲游戏信息组织）: 3
>
> GSRR（中国台湾|游戏软体分级管理办法）： 辅 12 级
>
> App Store 年龄分级 : 4+

</details>

<details>
<summary><b><font color="#8e4bbc">有关亚洲（繁中）服的信息</font></b></summary>

?> 亚服（繁中）可能需要**除中国大陆和日本外的亚洲地区 IP** 登陆，请提前做好相关准备。

</details>

<!-- tabs:start -->

#### **日服**

### 1.10.3 (2021/10/21)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/pjsekai/com.sega.pjsekai_1.10.3.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/pjsekai/com.sega.pjsekai_1.10.3.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/pjsekai/com.sega.pjsekai_1.10.3.apk)

#### **亚服 | 繁中**

### 1.7.3 (2021/10/25)

[三网通用 | 美西 Linode 节点](https://liusw.rytsu.org/pjsekai/pjsk_com.hermes.mk.asia_1.7.3.apk)

[移动最佳 | 香港 CMI 节点](https://hkcmi.rytsu.org/pjsekai/pjsk_com.hermes.mk.asia_1.7.3.apk)

[备用线路 | 东京 Linode 节点](https://litokyo.rytsu.org/pjsekai/pjsk_com.hermes.mk.asia_1.7.3.apk)

<!-- tabs:end -->

changelog.md
# 更新日志

见证本站历史的地方。

更新日志实际上是对过去一段时间内众多 commits 的总结，更详细的改动记录可以查看 [GitHub 上本项目的 commits](https://github.com/Lytsu/lowlight/commits/main) 。

## 1.x

### 2021 年 10 月

#### 2021/10/22

- 新增游戏：
  - Lanota；
  - Cytoid；
  - Dynamite；
  - 偶像梦幻祭 2 （ES2）；
- 更新游戏：
  - BanG Dream 日服 → 5.6.4；
  - D4DJ 国际服 → 2.5.3；
- 为降低运营成本，即日起本站不再提供 Phigros 和 Orzmic 的最新版本，仅留存历史版本。 

#### 2021/10/21 - 1.0.0

- 游戏更新：
  - Arcaea → 3.8.8；
  - 世界计划 多彩舞台（PRSK/PJSK） → 1.10.3
- 本站技术层面上的内容基本搭建完成，不出意外未来很长一段时间都不会进行大的改动，因此今天的版本正式 tag 成 1.0.0。

#### 2021/10/17

- 为所有游戏的下载链接增加了「适合的线路」标识；
  - 就是 移动/联通/电信/通用 这些；
- 新增以下游戏：
  - osu! (Windows Classic)；
  - osu! Lazer (Windows/macOS/Linux/Android)；
- 启用域名：[sega.sbs](https://sega.sbs)；
- 其他细节优化。

#### 2021/10/16

- 新增以下游戏：
  - Orzmic；
  - TAKUMI³；
- 更新以下游戏：
  - BanG Dream! 日服 → 5.6.1；
  - D4DJ 日服 → 2.5.6；
  - D4DJ 英文服 → 2.4.5；
  - LLSIF 日服/国际服 → 9.3；
- 新增[支持与感谢](support)页面；
- 页面重新排版；
- 文本细节打磨。

#### 2021/10/12

- 新增游戏：Sonolus；
- 为了节省侧边栏空间，将~~根本没人看的~~网站条款和政策挪到了导航栏；
- 新增[桌面端音游专页](desktop)；
- 增加[查看服务器状态](https://status.lowi.ro/status/)入口；
- 其他一些更新。

#### 2021/10/11

- 重新设计页面，更换主题色为 `#8e4bbc` ；
- 新增 Tab 样式，目前 Arcaea 和 Phigros 已应用本样式；
- 新域名：[lowlight.pages.dev](https://lowlight.pages.dev) ；
- 新增 GitHub repo 快速入口；
- **新增了站娘！！！由 [Kuriko](https://space.bilibili.com/7847180) 绘制；**
- Add [donating methods](donate) ；
- 其他一些改动。

#### 2021/10/07

- 由于新增了域名，本站名义正式更换至 「微光 Lowlight」；
- 因下载站名义更换，所有政策均有更新；
- 新增 Phigros 的数个旧版本；
- 增加[收录规则](inclusion)；
- 侧边栏排版优化；
- 技术细节信息优化；
- 文本瑕疵优化；
- 标点符号错用修改。

#### 2021/10/06

- 启用另一个域名 [nhe.lv](https://nhe.lv)

#### 2021/10/04

- 更新游戏：PJSK（日） 1.10.1

#### 2021/10/01

- 国庆快乐！本站历史上第一次大更（
- 新增以下游戏：
  - BanG Dream（日/简中）
  - D4DJ（日/国际）
  - PJSK（日/繁中）
  - Malody（Win/安卓）
  - LLSIF（日/国际/简中）
- 新增以下文档：
  - [SAI 教程](sai)
  - [Apple ID 注册教程](appleid)
- 新增[转载规范](repost)，同时更新了部分条款；
- 新增以下功能和特性：
  - 搜索（位于侧边栏）
  - 友链区域（位于侧边栏）
  - 离线优化（没网也能访问下载站）
  - 图片展示优化（图库画廊功能）
- 开放首页的问卷调查
- ~~移除了 Herobrine~~
- 祝各位国庆快乐！

### 2021 年 9 月

#### 2021/09/29

- 更新 Arcaea 3.8.6。

#### 2021/09/28

- 加了一台 Linode 机子，由此可以说本站可以正式开始稳定运营了。顺便准备准备明天更新的 Arcaea 。


sidebars.js
/**
 * Creating a sidebar enables you to:
 - create an ordered group of docs
 - render a sidebar for each doc of that group
 - provide next/previous navigation

 The sidebars can be generated from the filesystem, or explicitly defined here.

 Create as many sidebars as you want.
 */

// @ts-check

/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
const sidebars = {
    // By default, Docusaurus generates a sidebar from the docs folder structure
    tutorialSidebar: [
        "games",
        "desktop",
        "faq",
        {
            type: 'category',
            label: '站点信息',
            collapsed: true,
            items: [
                'sideinfo/about',
                'sideinfo/donate',
                'sideinfo/support',
            ],
        },
        "changelog",
    ],
    // But you can create a sidebar manually
    /*
    tutorialSidebar: [
      {
        type: 'category',
        label: 'Tutorial',
        items: ['hello'],
      },
    ],
     */
};

module.exports = sidebars;

src/pages/index.js
import React from 'react';
import clsx from 'clsx';
import Layout from '@theme/Layout';
import Link from '@docusaurus/Link';
import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
import styles from './index.module.css';
import HomepageFeatures from '../components/HomepageFeatures';

function HomepageHeader() {
  const {siteConfig} = useDocusaurusContext();
  return (
    <header className={clsx('hero hero--primary', styles.heroBanner)}>
      <div className="container">
        <p className="hero__subtitle">{siteConfig.tagline}</p>
        <div className={styles.buttons}>
          <Link
            className="button button--secondary button--lg"
            to="games">
            前往下载游戏
          </Link>
        </div>
      </div>
    </header>
  );
}

export default function Home() {
  const {siteConfig} = useDocusaurusContext();
  return (
    <Layout
      title={`首页`}
      description="Lowlight - A Faint Light">
      <HomepageHeader />
      <main>
        <HomepageFeatures />
      </main>
    </Layout>
  );
}


src/components/HomepageFeatures.js
import React from 'react';
import clsx from 'clsx';
import styles from './HomepageFeatures.module.css';

const FeatureList = [
  {
    title: '做多数人喜爱的下载站',
    description: (
      <>
        <p>作为音乐节奏游戏的下载分流之一，微光 Lowlight 希望能让更多人能够下载到更多的音乐节奏游戏。</p>
        <p>与此同时，我们也希望通过简单的文字描述去帮助更多人体验游戏的快乐。</p>
        <p>在微光 Lowlight 自由下载您想要的游戏吧！</p>
      </>
    ),
  },
  {
    title: '全球多个节点加速下载',
    description: (
      <>
        <p>微光 Lowlight 部署了数个遍布于全球各地的下载节点，这些节点均分配了足够的带宽。</p>
        <p>无论您使用什么网络运营商，在世界上的什么地方，我们都希望为您提供优质的服务。</p>
        <p><s>虽然有时候还是会宕机掉线就是了</s></p>
      </>
    ),
  },
  {
    title: '让所有玩家都从中受益',
    description: (
      <>
        <p>虽然不敢立下过于绝对的保证，但我们认为本站应该是公益性质且不求回报的。</p>
        <p>我们不仅提供了游戏的分流下载，还提供了其他相关文档，帮助您获得更好的游戏体验。</p>
        <p>这就是微光 Lowlight ，做一缕微弱却有意义的光。</p>
      </>
    ),
  },
];

function Feature({title, description}) {
  return (
    <div className={clsx('col col--4')}>
      <div className="text--center padding-horiz--md">
        <h3>{title}</h3>
        <p>{description}</p>
      </div>
    </div>
  );
}

export default function HomepageFeatures() {
  return (
    <section className={styles.features}>
      <div className="container">
        <div className="row">
          {FeatureList.map((props, idx) => (
            <Feature key={idx} {...props} />
          ))}
        </div>
      </div>
    </section>
  );
}


docusaurus.config.js
// @ts-check
// Note: type annotations allow type checking and IDEs autocompletion

const lightCodeTheme = require('prism-react-renderer/themes/github');
const darkCodeTheme = require('prism-react-renderer/themes/dracula');

/** @type {import('@docusaurus/types').Config} */
const config = {
    title: '微光 Lowlight',
    tagline: '提供一些音游下载的小站',
    url: 'https://lowi.ro',
    baseUrl: '/',
    onBrokenLinks: 'warn',
    onBrokenMarkdownLinks: 'warn',
    favicon: 'img/favicon.ico',
    i18n: {
        defaultLocale: "zh-Hans",
        locales: ["zh-Hans"],
    },
    organizationName: 'Lytsu', // Usually your GitHub org/user name.
    projectName: 'lowlight', // Usually your repo name.

    presets: [
        [
            '@docusaurus/preset-classic',
            /** @type {import('@docusaurus/preset-classic').Options} */
            ({
                docs: {
                    sidebarPath: require.resolve('./sidebars.js'),
                    // Please change this to your repo.
                    editUrl: 'https://github.com/Lytsu/lowlight/edit/main/',
                    routeBasePath: "/",
                    showLastUpdateTime: true,
                    showLastUpdateAuthor: true,
                },
                blog: {
                    path: 'blog',
                    blogTitle: 'Lowlight 博客',
                    blogSidebarTitle: '所有文章',
                    blogSidebarCount: 'ALL',
                    showReadingTime: true,
                    postsPerPage: 5,
                    feedOptions: {
                        type: 'all',
                        copyright: `Copyright © ${new Date().getFullYear()} Intelligence office of Sailoud LLC.`,
                    },
                    // Please change this to your repo.
                    editUrl: 'https://github.com/Lytsu/lowlight/edit/main/blog/',
                },
                theme: {
                    customCss: require.resolve('./src/css/custom.css'),
                },
            }),
        ],
    ],

    themeConfig:
    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
        ({
        announcementBar: {
            id: 'support_us',
            content: '使用微光 Lowlight 的服务，即代表您已知悉并同意本站的<a href="policies/tos">使用条款</a>和<a href="policies/privacy">隐私政策</a>。<br>本站目前处于公开测试运营阶段，欢迎填写<a href="https://wj.qq.com/s2/1091099/5c62">问卷调查</a>来支持我们的工作。',
            backgroundColor: '#a26ac8',
            textColor: '#fff',
            isCloseable: false,
        },
        navbar: {
            title: '微光 Lowlight',
            logo: {
                alt: 'Lowlight Logo',
                src: 'img/lowlight_maid.png',
            },
            items: [{
                    type: 'doc',
                    docId: 'games',
                    position: 'left',
                    label: 'A Faint Light',
                },
                { to: '/blog', label: '博客', position: 'left' },
                {
                    label: '更新日志',
                    to: 'changelog',
                    position: 'right',
                },
                {
                    href: 'https://github.com/Lytsu/lowlight',
                    label: 'GitHub',
                    position: 'right',
                },
                {
                    label: '哔哩哔哩',
                    href: 'https://space.bilibili.com/319171871',
                    position: 'right',
                }
            ],
        },
        footer: {
            style: 'dark',
            links: [{
                    title: '站点信息',
                    items: [{
                            label: '服务器状态',
                            href: 'https://status.lowi.ro/status/',
                        },
                        {
                            label: '更新日志',
                            to: 'changelog',
                        },
                    ],
                },
                {
                    title: '网站政策',
                    items: [{
                            label: '使用条款',
                            to: 'policies/tos',
                        },
                        {
                            label: '隐私政策',
                            to: 'policies/privacy',
                        },
                        {
                            label: '未成年人健康守护',
                            to: 'policies/minor',
                        },
                        {
                            label: '资源转载规范',
                            to: 'policies/repost',
                        },
                        {
                            label: '游戏收录标准',
                            to: 'policies/inclusion',
                        },
                    ]
                },
                {
                    title: '关注我们',
                    items: [{
                            label: '博客',
                            to: '/blog',
                        },
                        {
                            label: 'GitHub',
                            href: 'https://github.com/Lytsu/lowlight',
                        },
                        {
                            label: '哔哩哔哩',
                            href: 'https://space.bilibili.com/319171871',
                        },
                        {
                            label: 'Telegram',
                            href: 'https://t.me/lowi.ro',
                        },
                    ],
                },
                {
                    title: '友情链接',
                    items: [{
                            label: 'Konmai 音游下载站',
                            href: 'https://konmai.cn',
                        },
                        {
                            label: 'Rsplwe Blog',
                            href: 'https://www.rsplwe.com',
                        },
                        {
                            label: 'VicACG Community',
                            href: 'https://vikacg.com',
                        },
                        {
                            label: 'Karyl 工具箱',
                            href: 'https://bilibili.red',
                        },
                        {
                            label: '空',
                            href: 'https://minasan.xyz',
                        },
                        {
                            label: '空白白白白白',
                            href: 'https://thneden.github.io',
                        }
                    ],
                },
            ],
            copyright: `Copyright © ${new Date().getFullYear()} Intelligence office of Sailoud LLC. 使用 <a href="https://docusaurus.io/zh-CN">Docusaurus v2</a> 构建。<br>
            首页封面图 Pixiv ID: <a href="https://www.pixiv.net/artworks/79558921">79558921</a> | <a href="https://www.pixiv.net/artworks/82358027">82358027</a> `,
        },
        prism: {
            theme: lightCodeTheme,
            darkTheme: darkCodeTheme,
        },
    }),
};

module.exports = config;